### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by slichtzzz**
Update Russian translation

### CODE REVIEW

File: iTestServer/i18n/iTestServer-ru.ts [hunk 1]
```diff
@@ -2176,7 +2178,7 @@ Please choose a different group.</source>
     </message>
     <message>
         <source>Scalable Vector Graphics (*.svg);;All files (*.*)</source>
-        <translation>Файл масштабируемой векторной графики (*.svg);;Все файлы (*.*)</translation>
+        <translation>Scalable Vector Graphics (*.svg);;Все файлы (*.*)</translation>
```
Comment (michal-tomlein) [inline iTestServer/i18n/iTestServer-ru.ts:2181]: Wouldn’t it be better to keep the translated version?

File: iTestServer/i18n/iTestServer-ru.ts [hunk 2]
```diff
@@ -2212,7 +2214,7 @@ Please choose a different group.</source>
     </message>
     <message>
         <source>Scalable Vector Graphics (*.svg)</source>
-        <translation>Файл масштабируемой векторной графики (*.svg)</translation>
+        <translation>Scalable Vector Graphics (*.svg)</translation>
```
Comment (michal-tomlein) [inline iTestServer/i18n/iTestServer-ru.ts:2217]: Same here. What is the motivation for this change?

File: iTestServer/i18n/iTestServer-ru.ts (hunk 1 above)
Comment (slichtzzz) [reply iTestServer/i18n/iTestServer-ru.ts:2181]: Hi, Michal! It's just my opinion that translation of the abbreviation doesn't help much, when we deal with the file formats in "Save File" dialogs, and it's better to keep the good old English version, which shows exactly how the abbreviation is disclosed. Especially keeping in mind that different translators keep translating these abbreviations differently, which may lead to confusion sometimes. But it's only my opinion and of course I wouldn't be against if you do the way you would like it to be.

Comment (michal-tomlein) [reply iTestServer/i18n/iTestServer-ru.ts:2181]: I understand, but I think I’ve seen this translated in most open/save dialogs. Can you update your merge request to exclude this change or suggest a different Russian translation? Perhaps Файл [could be dropped](https://ru.wikipedia.org/wiki/SVG).

**Comment by michal-tomlein** [comment]
Thank you.
